libraries/math/src/vector3/coreHelpers.ts

```diff
@@ -123,6 +132,32 @@ export class Vector3Utils {
             z: clampNumber(v.z, limits?.min?.z ?? Number.MIN_SAFE_INTEGER, limits?.max?.z ?? Number.MAX_SAFE_INTEGER),
         };
     }
+
+    /**
+     * lerp
+     *
+     * Constructs a new vector using linear interpolation on each component from two vectors.
+     */
+    static lerp(a: Vector3, b: Vector3, t: number): Vector3 {
+        return {
+            x: a.x + (b.x - a.x) * t,
+            y: a.y + (b.y - a.y) * t,
+            z: a.z + (b.z - a.z) * t,
+        };
+    }
+
+    /**
+     * slerp
+     *
+     * Constructs a new vector using spherical linear interpolation on each component from two vectors.
+     */
+    static slerp(a: Vector3, b: Vector3, t: number): Vector3 {
+        const theta = Math.acos(Vector3Utils.dot(a, b));
+		const sinTheta = Math.sin(theta);
+		const ta = Math.sin((1.0 - t) * theta) / sinTheta;
+		const tb = Math.sin(t * theta) / sinTheta;
+		return Vector3Utils.add(Vector3Utils.scale(a, ta), Vector3Utils.scale(b, tb));
+    }
```

Comment:
Can you add the object oriented helpers for this in `vectorWrapper.ts`?

Reply:
Yep whoops I saw those but forgot to add them afterwards!